Chatkit/Data Simulation/DataSimulator+Events.swift

```diff
@@ -192,17 +192,33 @@ extension DataSimulator {
         return user
     }
     
-    private func createRoom(in context: NSManagedObjectContext, identifier: String, name: String, members: [UserEntity]) -> RoomEntity {
+    private func createRoom(in context: NSManagedObjectContext, identifier: String, userName: String, isSubscribedToExerciseRoutinesPlan: Bool = false, members: [UserEntity]) -> RoomEntity {
         let now = Date()
         
         let room = context.create(RoomEntity.self)
         room.identifier = identifier
-        room.name = name
         room.unreadCount = 0
         room.isPrivate = false
         room.createdAt = now
         room.updatedAt = now
         
+        if isSubscribedToExerciseRoutinesPlan {
+            let planName = "Exercise Routine"
+            let userData = ["planID" : "exercise-basic",
+                            "planName" : planName]
+            
+            room.name = "\(userName) - \(planName) Plan"
+            room.userData = UserDataSerializer.serialize(userData: userData)
+        }
+        else {
+            let planName = "Nutrition"
+            let userData = ["planID" : "routine-basic",
+                            "planName" : planName]
+            
+            room.name = "\(userName) - \(planName) Plan"
```

Comment:
I think it would be more realistic to have just the `planName` in the room name field, and have the implementor combine the user name and plan name to come up with the display string. We expect dynamically computed conversation titles to be a common case, especially in one-to-one use case.

Reply:
No problem. Should we drop user name from the name of the room as well?

Reply:
No problem.

Reply:
Done